pages/api/task/create.js

```diff
@@ -0,0 +1,24 @@
+import Task from '../../../src/lib/models/Task';
+import { dbConnect } from '../../../src/lib/db/database';
+
+export default async function handler(req, res) {
+	if (req.method === 'POST') {
+		const newTaskData = JSON.parse(req.body);
+		await dbConnect();
+
+		const newTask = await Task.create({
+			name: newTaskData.name,
+			done: newTaskData.done,
+			dueDate: newTaskData.dueDate,
+			doneDate: newTaskData.doneDate,
+			startDate: newTaskData.startDate,
+		});
+		console.log(req.body);
```

Comment:
Please delete console.log statements that are used for debugging during development.

Reply:
I thought these where for Serverside visualization. Thanks again